EXO12.pas

```diff
@@ -0,0 +1,35 @@
+
+
+//EXO12: Proposer un Algo puis un programme Pascal permettant de saisir n entiers 
+// et d'afficher le plus grand et le plus petit parmi eux et la moyenne de tous les entiers, 
+//n étant saisis
+
+Program EXO12;
+Var 
+    i,n,a,max,min,som : integer;
+    moy :  longint;
+Begin
+    writeln('Entrer le nombre de fois :');
+    readln(n);
+    min := 0;
```

Comment:
Revoir les initialisations au lieu de 0 mais en a, ensuite executer une itération au dehors et demarrer le décompte à 2.
```suggestion
    min := a;
    max:=a;
    som:=a;
    demarrer le 1ere itération : Wln('Entrer a:'); Readln(a); 
    Enfin decompter à partir de  :
    for i:=2 to n 
    begin 
       bloc d'inst
    end;
```